spec/index.bs

```diff
@@ -8075,6 +8077,14 @@ dictionary GPUFragmentState
                     : {{GPUTextureSampleType/"uint"}}
```

Comment:
Line 8063 assumes that there can only be one `|output|` at this location, but now there can be 2. The wording needs updates to handle this: either loop over the 2, or duplicate the whole section for blend_src=0 and blend_src=1.